scripts/app.ts

```diff
@@ -27,6 +29,8 @@ const dbPromise = (async () => {
 })();
 
 const app = express();
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
 
```

Comment:
TODO:
- Add all songs path app.get("/api/songs")
- Add path to show edit template app.get("/edit")

- Add user login flows